examples/asyncio.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Companion code to the blog post "Integrating Kafka With Python
+# Asyncio Web Applications"
+# https://www.confluent.io/blog/[path-to-blog-post]
+
+# Example Siege [https://github.com/JoeDog/siege] test:
+# siege -c 400 -r 200 'http://localhost:8000/items1 POST {"name":"testuser"}'
+
+import asyncio
+import confluent_kafka
+from confluent_kafka import KafkaException
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from time import time
+from threading import Thread
+import uvicorn
+
+
+class AIOProducer:
+    def __init__(self, configs, loop=None):
+        self._loop = loop or asyncio.get_event_loop()
+        self._producer = confluent_kafka.Producer(configs)
+        self._cancelled = False
+        self._poll_thread = Thread(target=self._poll_loop)
+        self._poll_thread.start()
+
+    def _poll_loop(self):
+        while not self._cancelled:
+            self._producer.poll(0.1)
+
+    def close(self):
+        self._cancelled = True
+        self._poll_thread.join()
+
+    def produce(self, topic, value):
+        result = self._loop.create_future()
+
+        def ack(err, msg):
+            if err:
+                self._loop.call_soon_threadsafe(result.set_exception, KafkaException(err))
+            else:
+                self._loop.call_soon_threadsafe(result.set_result, msg)
+        self._producer.produce(topic, value, on_delivery=ack)
+        return result
+
+    def produce2(self, topic, value, on_delivery):
```

Comment:
two produce methods with the same name (numbers dont count) definitely need doc strings